### DISCUSSION TITLE
Which language do you think is the best at this time?

### DISCUSSION THREAD

**Discussion by v1sh-afk** [community/community#134780] [Programming Help] [answered]
### Body

Which language do you think is the best at this time? 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by HallowSiddharth** [comment]
Hi Vishal,
I personally feel that Python is the best programming language out there.
It's very easy to get started with, the scope for it is a lot

Especially with the rise of AI/ML, I think Python is the way to go for beginners

**Comment by ebndev** [comment, marked as answer]
Please see [this comment](https://github.com/orgs/community/discussions/134781#discussioncomment-10246657) for why this discussions was closed and locked.
